pkg/kubectl/resource/result.go

```diff
@@ -236,8 +242,12 @@ func AsVersionedObject(infos []*Info, forceList bool, version schema.GroupVersio
 	if len(objects) == 1 && !forceList {
 		object = objects[0]
 	} else {
-		object = &api.List{Items: objects}
-		converted, err := TryConvert(api.Scheme, object, version, api.Registry.GroupOrDie(api.GroupName).GroupVersion)
+		e := []runtime.RawExtension{}
+		for _, o := range objects {
+			e = append(e, runtime.RawExtension{Object: o})
+		}
+		object = &v1.List{Items: e}
+		converted, err := TryConvert(scheme.Scheme, object, version, schema.GroupVersion{"", "v1"})
```

Comment:
Is `object` supposed to be an internal versioned object? If so , scheme.Scheme won't be able to do the conversion. It only knows about external versions.